### ISSUE TITLE
Default parameter value taken from variable is applied only for last parameter

### ISSUE THREAD

**Issue by ignis32** [GilesBathgate/RapCAD#14]
Something is kinda strange with assigning default values taken from variable.
When I try to assign default value to the module parameters,
this value is assigned only to the last parameter.
RapCad version is 0.9.8

Code:

 
```
default=999999;

module section(x=default,y=default,z=default)
{
	echo("x");
	echo(x);

	echo("y");
	echo(y);

	echo("z");
	echo(z);
 
}
 

section();
```
 
 

Output:

 
```
ECHO: "x"
ECHO: undef
ECHO: "y"
ECHO: undef
ECHO: "z"
ECHO: 999999
Warning: no top level object.
```

**Comment by GilesBathgate** [comment]
Thanks I will add this to my test cases and look for a fix.

**Comment by GilesBathgate** [comment]
fixed in commit 0ad841f865d334af95a1bba9df1156968fda3eb5
